### ISSUE TITLE
[yangster-docker] can't open files

### ISSUE THREAD

**Issue by marcdumais-work** [theia-ide/theia-apps#33]
When I try to open files in Theia, each time it fails and I get this exception in the frontend:
```
reflection_utils.js:33 Uncaught (in promise) Error: The number of constructor arguments in the derived class t must be >= than the number of constructor arguments of its base class.
    at u (reflection_utils.js:33)
    at Object.t.getDependencies (reflection_utils.js:11)
    at planner.js:106
    at Array.forEach (<anonymous>)
    at e (planner.js:94)
    at Object.t.plan (planner.js:129)
    at container.js:316
    at e._get (container.js:309)
    at e.get (container.js:228)
    at e.editorProvider (monaco-frontend-module.ts:53)
```

This is from freshly built images on my machine - I have not tried with the image deployed on dockerhub. 

Tried also with the more basic theia-docker image, and that one does not seem to have that issue. Maybe related to the yangster extension?

**Comment by marcdumais-work** [comment]
Probably because the currently published ```latest``` version of the yang extension is a bit old and incompatible with the ```0.3.8``` version of Theia extensions. see https://github.com/theia-ide/yangster/issues/58

**Comment by marcdumais-work** [comment]
Fix through theia-ide/yangster#58
